blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -558,13 +519,13 @@ export const actions = {
                 // We're only interested in this event if we're already managing branch state
                 // associated with this multi-branch job.
                 if (!multibranchPipelines[pipelineName]) {
-                    return;
+                    return Promise.resolve(null);
```

Comment:
... same would be true for any other places you're explicitly creating promises, of course